scripts/qa_crowbarsetup.sh

```diff
@@ -3388,9 +3388,14 @@ function get_manila_service_instance_details
 function addfloatingip
 {
     local instanceid=$1
-    nova floating-ip-create | tee floating-ip-create.out
-    floatingip=$(perl -ne "if(/\d+\.\d+\.\d+\.\d+/){print \$&}" floating-ip-create.out)
-    nova add-floating-ip "$instanceid" "$floatingip"
+    if iscloudver 8plus; then
```

Comment:
I'm reasonably sure the floating ip commands were available in the Newton series of python-openstackclient releases, so you could switch this to 7plus